bftengine/src/preprocessor/RequestProcessingState.cpp

```diff
@@ -77,15 +77,6 @@ uint32_t RequestProcessingState::sizeOfPreProcessResultData() const {
   return sizeof(OperationResult) + sizeof(clientId_) + sizeof(reqSeqNum_);
 }
 
-void RequestProcessingState::setupPreProcessResultData(OperationResult preProcessResult) {
-  memcpy(&primaryPreProcessResultData_, &preProcessResult, sizeof(preProcessResult));
-  primaryPreProcessResultData_ += sizeof(preProcessResult);
-  memcpy(&primaryPreProcessResultData_, &clientId_, sizeof(clientId_));
-  primaryPreProcessResultData_ += sizeof(clientId_);
-  memcpy(&primaryPreProcessResultData_, &reqSeqNum_, sizeof(reqSeqNum_));
-  primaryPreProcessResultLen_ = sizeOfPreProcessResultData();
-}
-
```

Comment:
Why did you remove this code?

Reply:
added back with change. Earlier & address was passed, and this caused issue in copying the data back to clientRequest.